crates/orderbook/openapi.yml

```diff
@@ -965,3 +989,18 @@ components:
             the fee after this expiration date. Encoded as ISO 8601 UTC.
           type: string
           example: "1985-03-10T18:35:18.814523Z"
+    SolverCompetitionResponse:
+      type: object
```

Comment:
Felix mentioned that we should likely also include the transaction hash in the settlement competition response. This will make it easier to understand the auction that lead to a specific solver winning.

I think Felix will connect you with someone who plans operating an external solver. Since this API is very important for external solvers, we should understand their use-case a bit and try to shape the endpoint to fit it as best as possible.

Reply:
Looks like we don't need the route to be available by transaction hash. Resolving thread for now.